OpenCVExample/OpenCVExample/MainPage.xaml.h

```diff
@@ -0,0 +1,28 @@
+﻿//
+// MainPage.xaml.h
+// Declaration of the MainPage class.
+//
+
+#pragma once
+
+#include "MainPage.g.h"
+#include <opencv2\core\core.hpp>
+
+namespace OpenCVExample
+{
+	/// <summary>
+	/// An empty page that can be used on its own or navigated to within a Frame.
+	/// </summary>
+	public ref class MainPage sealed
+	{
+	public:
+		MainPage();
+	private:
+		cv::Mat Lena;
```

Comment:
Usually, the coding standard is using an underscore and camel notation for data members (for example; _lena). Also, what does Lena mean?